core/modules/flowgen.cc

```diff
@@ -200,6 +200,13 @@ pb_error_t FlowGen::ProcessArguments(const bess::pb::FlowGenArg &arg) {
     quick_rampup_ = 1;
   }
 
+  if (arg.burst() <= bess::PacketBatch::kMaxBurst) {
```

Comment:
`FlowGen` already has a `update()` command. How about using that command? (I am fine with adding a new command if you prefer)